jmclient/jmclient/wallet.py

```diff
@@ -331,7 +332,8 @@ def sign_tx(self, tx, scripts, **kwargs):
             assert amount > 0
             path = self.script_to_path(script)
             privkey, engine = self._get_priv_from_path(path)
-            engine.sign_transaction(tx, index, privkey, amount, **kwargs)
+            tx = btc.deserialize(engine.sign_transaction(tx, index, privkey,
```

Comment:
if `engine.sign_transaction` receives a deserialized tx I think it should also return a deserialized tx.